py-rse/package-py.Rmd

```diff
@@ -482,450 +173,562 @@ without affecting the main installation (Figure \@ref(fig:py-rse-package-py-vir
 knitr::include_graphics("figures/FIXME.png")
 ```
 
-We can create and manage virtual environments using a tool called `virtualenv`.
-To install it,
-run `pip install virtualenv`.
-Once we have done that,
-we can create a new virtual environment called `test` by running:
+We can create and manage virtual environments using `conda`,
+the same tool we used to install packages earlier.
+We can create a new virtual environment called `zipfs-law` with the `conda create` command
+and specify the environment name with the `-n` or `--name` flag:
 
 ```shell
-$ virtualenv test
+$ conda create -n zipfs-law
 ```
+
 ```text
-Using base prefix '/Users/pterry/anaconda3'
-New python executable in /Users/pterry/test/bin/python
-Installing setuptools, pip, wheel...
-done.
+Collecting package metadata (current_repodata.json): done
+Solving environment: done
+
+## Package Plan ##
+
+  environment location: /home/amira/anaconda3/envs/zipfs-law
+
+
+
+Proceed ([y]/n)? y
+
+Preparing transaction: done
+Verifying transaction: done
+Executing transaction: done
+#
+# To activate this environment, use
+#
+#     $ conda activate zipfs-law
+#
+# To deactivate an active environment, use
+#
+#     $ conda deactivate
 ```
 
-`virtualenv` creates a new directory called `test`,
+`conda` creates a new directory called `zipfs-law`,
 which contains subdirectories called `bin`, `lib`, and so on—everything
 needed for a minimal Python installation.
 Crucially,
-`test/bin/python` checks for packages in `test/lib` *before* checking the system-wide install.
+`zipfs-law/bin/python` checks for packages in these directories instead of checking the system-wide install.
```

Comment:
Might be clearer to put the whole python path? 

"`/home/amira/anaconda3/envs/zipfs-law/bin/python` checks ..."